modules/virt-deleting-templates-from-custom-namespace.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deleting-templates-from-custom-namespace.adoc
+
+[id="virt-deleting-templates-from-custom-namespace_{context}"]
+= Deleting templates from a custom namespace
+
+When you delete Red Hat templates that are deployed to a custom namespace, they are regenerated. To avoid regenerating the templates, replace the existing value for the custom namespace with a different value. This causes the templates to be deployed to the custom namespace with the new value. You can now delete the templates from the original custom namespace.
+
+.Procedure
+
+. Open the `hco` CR to edit the file.
++
+[source,terminal]
+----
+$ oc edit hco
```

Comment:
Needs to specify NS `openshift-cnv` here too.

Reply:
In the video meet that I had with @ksimon1 and when he walked through the steps, he explicitly mentioned that we are editing the hco custom resource. Therefore, I am asking him for clarification.

Reply:
@gouyang - Please confirm. Is this the correct command? If not, please provide the command syntax.
`$ oc edit hco -n openshift-cnv kubevirt-hyperconverged`

Reply:
> @gouyang - Please confirm. Is this the correct command? If not, please provide the command syntax. `$ oc edit hco -n openshift-cnv kubevirt-hyperconverged`

@lmandavi this is correct for edit command

Reply:
Updated the command syntax for the edit command.